### ISSUE TITLE
preference for no sound effects is lost between sessions

### ISSUE THREAD

**Issue by therentabrain** [thomasmeadows/citibank-van#6]
Unchecking the preference 'play sound effects' not only results in a sound effect as you return to the main menu (tacky :) ) but it also does not remember this preference for the next app use.

**Comment by therentabrain** [comment]
(I'm assuming this is the app downloadable from Citi which installs itself as 'github')

**Comment by krbvroc1** [comment]
This is a third party command line app that has nothing to do with the Citi provided app which is based on outdated and unsupported Flash technology. Your issue should be reported to Citi, not here.

**Comment by therentabrain** [comment]
OK thanks, FWIW, they just recently released a brand new rewrite which appears to avoid using Flash.  It announces itself not as Citibank but as Github (yes, the one downloadable directly from citi), and that brought me here.  Best!

**Comment by krbvroc1** [comment]
I did not know that, I will checkout the new version. Glad you mentioned it. Thanks.

**Comment by krbvroc1** [comment]
@therentabrain - I did check out that 'new rewrite'. It lasted for just a few weeks, then stopped working, and now Citi has removed it from their site. At least the online one now works without flash.

![Selection_013](https://user-images.githubusercontent.com/1335118/94732849-d44e0e00-0334-11eb-824e-35523da75a1e.png)

**Comment by therentabrain** [comment]
Yeah, it's been disabled for a few weeks now.  But yes, it works fine within the website, better than it used to...

**Comment by wicadmin** [comment]
But this project no longer works with it?

**Comment by krbvroc1** [comment]
Nope, login fails...this project is dead.
